Documentation~/cleanShotGrid.md

```diff
@@ -0,0 +1,17 @@
+## Uninstall existing ShotGrid Desktop apps
+
+Before installing a newer version of ShotGrid Desktop, it is recommended to remove
+the previously installed, older version, of ShotGrid Desktop.
+
+To uninstall ShotGrid Desktop:
+
+- On Mac: Applications → Utilities → Uninstall ShotGrid Desktop
+- On Windows: Add/Remove programs → Search for ShotGrid Desktop
+
+## Clear ShotGrid Desktop cache
+
+If you edit Pipeline Configuration, or a ShotGrid setting, but do not see
+ the effects of this change, try to clear the ShotGrid Desktop cache.
```

Comment:
"If you edit a Pipeline Configuration or a ShotGrid setting but do not see
 the effects of this change, try to clear the ShotGrid Desktop cache."